portfolio/src/main/webapp/index.html

```diff
@@ -51,8 +51,80 @@ <h2>About Me</h2>
             </div>
         </div>
       </div>
+
+      <br>
+      <br>
+      <br>
+      <br>
+      <br>
+      <br>
+      <hr>
+
       <div id="work"<p>
         <h2>Work</h2>
+
+        <div class="main">
+
+            <div class="content">
+                <img src="/w3images/p3.jpg" alt="Bear" style="width:100%">
+                <h3>UCSD Triton</h3>
+                <p class="justified-text">
+                    I am currently a student at UC San Diego, majoring in Computer Engineering. In the field of 
+                    computer science, I have taken classes on topics such as Software Engineering, Data Structures,
+                    and Algorithm Analysis. Moreover, I have taken courses in the field of electrical engineering, 
+                    such as Analog Design and Linear Systems. 
+                </p>
+            </div>
+
+            <!-- Portfolio Gallery Grid -->
+            <div class="row">
+                <div class="column">
+                    <div class="content">
+                        <img src="/w3images/mountains.jpg" alt="Mountains" style="width:100%">
```

Comment:
Nit: The image style seems redundant since it is specified in img css.

Reply:
Fixed by removing the style components in img tags, so all img style is in CSS